websocket/ratelimit.go

```diff
@@ -7,6 +7,55 @@ import (
 	"github.com/andersfylling/disgord/websocket/cmd"
 )
 
+type UniqueStringSlice struct {
```

Comment:
Would need some documenting that this is protected behind a mutex. Someone else might see this struct and think "Oh wow, I'm going to be lazy and use this without realising the potential for performance problems or maybe even deadlocks.